languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -59540,6 +59540,64 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <category id="TYPOGRAPHY" name="Typografie">
+        <rulegroup id="SPACE_AFTER_BEFORE SLASH" name="Leerzeichen vor und nach Slash" default="temp_off"
+            <rule id="SPACE_AFTER_SLASH" name="Leerzeichen nach Slash">
+                <antipattern>
+                    <token regexp="yes">.*[\wüöäß].*</token>
+                    <token spacebefore="yes">/</token>
+                    <token regexp="yes" spacebefore="yes">.*[\wüöäß].*</token>
+                </antipattern>
+                <antipattern>
+                    <token regexp="yes">.*[\wüöäß].*</token>
+                    <token spacebefore="no">/</token>
+                    <token regexp="yes" spacebefore="no">.*[\wüöäß].*</token>
+                </antipattern>
+                <pattern>
+                    <marker>
+                        <token regexp="yes">.*[\wüöäß].*</token>
+                        <token spacebefore="no">/</token>
+                        <token regexp="yes" spacebefore="yes">.*[\wüöäß].*</token>
+                    </marker>
+                </pattern>
+                <message>Um Unregelmäßigkeiten zu vermeiden, wird empfohlen keine Leerzeichen vor und nach / zu verwenden. Alternativ kann auch jeweils ein Leerzeichen vor und nach / stehen.</message>
+                <suggestion>/</suggestion>
+                <suggestion> / </suggestion>
+                <example correction="">Ich ruf dich unter der Nummer <marker>010/ 5456846</marker> an.</example>
+                <example>Ich ruf dich unter der Nummer 010/5456846 an.</example>
+                <example>Ich ruf dich unter der Nummer 010 / 5456846 an.</example>
+                <example correction="">Ich mag <marker>Hunde/ Katzen</marker> nicht.</example>
+                <example>Ich mag Hunde/Katzen nicht.</example>
+                <example>Ich mag Hunde / Katzen nicht.</example>
+            </rule>
+            <rule id="SPACE_BEFORE_SLASH" name="Leerzeichen vor Slash">
+                <antipattern>
+                    <token regexp="yes">.*[\wüöäß].*</token>
+                    <token spacebefore="yes">/</token>
+                    <token regexp="yes" spacebefore="yes">.*[\wüöäß].*</token>
+                </antipattern>
+                <antipattern>
+                    <token regexp="yes">.*[\wüöäß].*</token>
+                    <token spacebefore="no">/</token>
+                    <token regexp="yes" spacebefore="no">.*[\wüöäß].*</token>
+                </antipattern>
+                <pattern>
+                    <marker>
+                        <token regexp="yes">.*[\wüöäß].*</token>
+                        <token spacebefore="yes">/</token>
+                        <token regexp="yes" spacebefore="no">.*[\wüöäß].*</token>
+                    </marker>
+                </pattern>
+                <message>Um Unregelmäßigkeiten zu vermeiden, wird empfohlen Leerzeichen vor und nach / zu verwenden. Alternativ können sie auch komplett ausgelassen werden.</message>
```

Comment:
In der Message bitte nicht einfach / schreiben, sondern "/" oder, noch besser, Schrägstrich.
Die beiden Messages widersprechen sich übrigens. DIN-Norm (frag mich nicht welche) sagt: keine Leerzeichen, also ist das "empfohlen". Den Teil mit "Unregelmäßigkeiten" würde ich rausnehmen.
Sollten sich deine Korrekturen im Branch nicht automatisch hierher transferieren, schließe einfach diesen PR und mach einen neuen – dann können auch gleich die neuen Tests drüberlaufen.